ios/Exponent.xcodeproj/project.pbxproj

```diff
@@ -2963,9 +2973,11 @@
 					"-Xlinker",
 					"-no_objc_category_merging",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = "host.exp.$(PRODUCT_NAME:rfc1034identifier)";
+				PRODUCT_BUNDLE_IDENTIFIER = dev.expo.Payments;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				PROVISIONING_PROFILE_SPECIFIER = "";
+				PROVISIONING_PROFILE = "$(APP_PROVISIONING_PROFILE)";
+				PROVISIONING_PROFILE_SPECIFIER = "Expo Payments Lab Development";
```

Comment:
We'll probably need to figure out a way to be able to test Ad Hoc Expo Client projects with custom modifications easily